vignettes/dust.Rmd

```diff
@@ -210,12 +210,12 @@ model$state(c(S = 1L, R = 3L))
 
 Again, this copies names from the index if they are present.
 
-In order to run the simulation beginning-to-end, we use `dust::dust_simulate()`; this function is purely a convenience function and the intention is that you would write your own (see [`mcstate`](https://mrc-ide.github.io/mcstate) for example).
+In order to run the simulation beginning-to-end, we use `dust::dust_iterate()`; this function is purely a convenience function and the intention is that you would write your own (see [`mcstate`](https://mrc-ide.github.io/mcstate) for example).
```

Comment:
Do we need to add `dust_simulate()` to the vignette? Or is that premature

Reply:
Definitely it should be added but perhaps we should go through and refresh the whole thing once I can drag this through mcstate and into sircovid2?